### DISCUSSION TITLE
Welcome to DocTalk Discussions!

### DISCUSSION THREAD

**Discussion by ItsRoy69** [ItsRoy69/DocTalk#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other contributor of this website. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other developers.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ItsRoy69** [comment]
We can first build the doctors page, then we can build a page which can help in one-to-one conversation like other social-media platforms without breaching of personal information like numbers.

**Comment by fduschka** [comment]
@ItsRoy69  Is there a database that can be used for testing?

**Reply by ItsRoy69** [reply]
There won't be any database given for testing but if you are using any database then do connect it with .env file and add another "dotenv" which'll show the contents of .env file without uri
